api/api.yaml

```diff
@@ -51,6 +51,10 @@ paths:
                   type: string
                   description: The provider that this app connects to.
                   example: salesforce
+                enableCsrf:
```

Comment:
Can you make this `enableCSRF`? (Since CSRF is actually an acronym, so it should be all caps, as opposed to `Id`, which is an abbreviation not an acronym). Also you accidentally left description and example empty.

Reply:
if this can be toggled from the browser, what security does it actually provide?

Reply:
@eberle1080 see discussion in #eng-discuss

Reply:
Another thing I realized, "CSRF" is Cross-site request forgery , so enableCSRF sounds like we are enabling the attach. We are actually enabling protection from the attack. So maybe `enableCSRFProtection` or something like that.

Reply:
@laurenzlong I left the description & example empty on purpose because this could be a security threat. If we describe it openly to all users in our documentation what this flag does, then using the flag defeats the purpose I feel. 

Let me know how you feel about it, I will add it otherwise.

Reply:
Then just remove the `description` and `example` lines, otherwise this is not a valid OpenAPI definition (as seen by the failed linter).